core/server/api/v2/utils/serializers/input/utils/url.js

```diff
@@ -0,0 +1,67 @@
+const {absoluteToRelative, getBlogUrl, STATIC_IMAGE_URL_PREFIX} = require('../../../../../../services/url/utils');
+
+function handleImageUrl(imageUrl) {
```

Comment:
We don't seem to need to hold a context of `this` and all the other functions in this module are declared with arrow notation. Let's keep it consistent?